requirements.txt

```diff
@@ -24,12 +24,12 @@ dealer
 flask-collect
 flake8
 passlib
-twill
+https://github.com/alexandre/twill/tarball/master
```

Comment:
I'd like to pass this repo to the QuokkaProject  as a "fork" of the original. For now, I've sent an email to the author about to create a mirror on GitHub or something similar.